templates/ovncontroller/bin/readiness.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# Function to handle errors and exit with a message
+error_exit() {
+    echo "$1"
+    exit 1
+}
+
+# Check if ovn-controller is connected to the OVN SB database
+check_ovn_controller_connection() {
+    output=$(ovn-appctl -t ovn-controller connection-status)
+    if [ $? -ne 0 ]; then
+        error_exit "ERROR - Failed to get connection status from ovn-controller"
+    fi
+    if [ "$output" != "connected" ]; then
+        error_exit "ERROR - OVN-Controller is not connected to OVN SB database"
+    fi
+}
+
+# Check if ovsdb-server is running and list its databases
+check_ovsdb_server() {
+    ovsdbserverpidfile=$(find /var/run/openvswitch -name "ovsdb-server.pid")
```

Comment:
do we really need to call `find` to find the file? Can't it be a hardcoded name since we probably already know the exact location?

Reply:
done, after checking that file is same path on dev env and a real one.